packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaEditor.tsx

```diff
@@ -56,10 +117,29 @@ export default function TableColumnSchemaEditor(
       />
     )
   }
+
   return (
-    <TableColumnSchemaForm
-      entityType={convertToEntityType(entity.concreteType)}
-      initialData={queryResultBundle?.responseBody?.columnModels}
-    />
+    <>
+      <TableColumnSchemaForm
+        ref={formRef}
+        entityType={convertToEntityType(entity.concreteType)}
+        initialData={queryResultBundle?.responseBody?.columnModels}
+        onSubmit={formData => {
+          onSubmit(formData)
+        }}
+      />
+      <Button
+        variant="contained"
+        color="primary"
+        onClick={() => {
+          if (formRef.current) {
+            formRef.current.submit()
+          }
+        }}
+        sx={{ my: 2 }}
+      >
+        Save
+      </Button>
```

Comment:
This save button is kind of a placeholder--refactoring will come later as we figure out exactly how we want to slot this into new/existing modals. For now, it lets our integration tests pass.